examples/servers/reading-list/.kube.yml

```diff
@@ -0,0 +1,97 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: reading-list-server
+spec:
+  minReadySeconds: 30
+  replicas: 5
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 2
+  template:
+    metadata:
+      labels:
+        app: reading-list-server
+    spec:
+      terminationGracePeriodSeconds: 60
+      volumes:
+      - name: nyt-reading-list-tls 
+        secret:
+          secretName: nyt-reading-list-tls
+      containers:
+        - name: esp
+          image: "gcr.io/endpoints-release/endpoints-runtime:1.3.0"
+          args: [
+            "-S", "443",
```

Comment:
This is a working example, it just requires the drone-gke plugin (which is called out in the readme) to complete & launch via that kube.yaml. The only missing piece here is a `GOOGLE_CREDENTIALS` secret.

The idea is that this example represents what one of our gRPC repos looks like inside NYT (I can send you an internal link for example and you can marvel at the similarities ;) ).

Also, we're looking into using `lego` as a real/production solution for services exposed publicly as Venafi does not work (not a _trusted_ CA from an App Engine POV).